From e51c476a240ffe6dfde53777a7c9675520abd29a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Sat, 30 Jul 2011 22:26:54 +0800
Subject: [PATCH] Revert "USB: gadget: Disable RNDIS function by default if
 CONFIG_USB_ANDROID_RNDIS is set"

This reverts commit cb7536857381c2eb10ecb4c1d14d2ef274baa75c.
---
 drivers/usb/gadget/f_rndis.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/usb/gadget/f_rndis.c b/drivers/usb/gadget/f_rndis.c
index 67a923fce6f0..af67a0096e09 100644
--- a/drivers/usb/gadget/f_rndis.c
+++ b/drivers/usb/gadget/f_rndis.c
@@ -826,11 +826,6 @@ int __init rndis_bind_config(struct usb_configuration *c, u8 ethaddr[ETH_ALEN])
 	rndis->port.func.setup = rndis_setup;
 	rndis->port.func.disable = rndis_disable;
 
-#ifdef CONFIG_USB_ANDROID_RNDIS
-	/* start disabled */
-	rndis->port.func.hidden = 1;
-#endif
-
 	status = usb_add_function(c, &rndis->port.func);
 	if (status) {
 		kfree(rndis);
-- 
2.35.3

